scannerazure.properties

```diff
@@ -6,8 +6,13 @@ organizationUrl=https://www.sonarsource.com
 homepageUrl=https://docs.sonarqube.org/latest/analysis/scan/sonarscanner-for-azure-devops/
 issueTrackerUrl=https://jira.sonarsource.com/browse/VSTS/issues
 sourcesUrl=https://github.com/SonarSource/sonar-scanner-vsts
-archivedVersions=4.7,4.7.1,4.7.2,4.8,4.8.1,4.9,4.10,4.11,4.12,4.16,4.17,4.18,4.19,4.20,4.21,4.22,4.23,4.23.1,5.0.0,5.0.1,5.1.1,5.2.0,5.3.0,5.4.0,5.5.0,5.6.0,5.6.1
-publicVersions=5.7.0
+archivedVersions=4.7,4.7.1,4.7.2,4.8,4.8.1,4.9,4.10,4.11,4.12,4.16,4.17,4.18,4.19,4.20,4.21,4.22,4.23,4.23.1,5.0.0,5.0.1,5.1.1,5.2.0,5.3.0,5.4.0,5.5.0,5.6.0,5.6.1,5.7.0
```

Comment:
Is this list of version unlimited in terms of numbers inside ? :)

Reply:
Good question I guess we'll find out someday...eventually ^^